README.md

```diff
@@ -1,6 +1,7 @@
 ## Ruby docs samples
 
-Kokoro: [![Kokoro](https://storage.googleapis.com/cloud-devrel-kokoro-resources/ruby/ruby-docs-samples/system_tests-ubuntu.png)](https://fusion.corp.google.com/projectanalysis/current/KOKORO/prod:cloud-devrel%2Fruby%2Fruby-docs-samples%2Fsystem_tests)
```

Comment:
I think the intention is for this to be the AND of the other two jobs? But, sometimes the permissions get messed up on the GCS object -- I'm not sure why.

Reply:
Last week, I updated the kokoro build to separate the results for each and generate separate HTML output.

Reply:
I believe the older configuration was failing because the badge was being generated within the presubmit.